crates/grafbase-local-backend/src/dev.rs

```diff
@@ -82,231 +90,50 @@ pub async fn start(
         }
     };
 
+    let (reload_sender, reload_receiver) = mpsc::channel::<(String, Config)>(1);
+
     let server_config = ServerConfig {
         listen_addr: Some(listen_address),
         config_path: None,
         config_hot_reload: false,
```

Comment:
This will be removed and `config_path` will be used to determine whether to turn on the gateway's internal hot reloading for a static schema (which only currently changes rate limiter options), I had done this already but apparently have reverted it at some point